layouts/partials/misc/contacts.html

```diff
@@ -24,8 +24,8 @@
             {{/* TODO: discuss */}}
         </div>
         <div class="col col-md-7">
-            <div class="overlay-card h-100 p-4 p-md-5 p-lg-6">
-                <p class="pb-4">You can also get in touch with us directly via this form.</p>
+            <div class="overlay-card h-100 p-4 p-md-5 p-lg-5 ">
+                <h2 class="mb-4">Have a Question?</h2>
```

Comment:
maybe `Got a Question?` is better?

Reply:
I prefer the formal style, `Have a question?`. Do you think it's better to use informal style?

Reply:
I'm fine with either style, I just don't thin `Have a question?` poses a proper english question.

If we want to stick with the formal way, how about `Do you have a question?`

Reply:
I thought about it, and it's too formal and too long. I wanted to use something that is more concise and to the point. That is also more formal than `got a question`.  Plus, this shorter version sounds friendlier and should have better customer engagement than the full sentence.

We can also ignore this change and leave the older text. Please let me know how you would like to proceed.

Reply:
how about just `Need help?` ?

Reply:
I think that one can work as well. It opens up the door to other topics that customers can bring up, but that shouldn't be a problem.

I will update it.